src/contracts/facilitators/gsm/feeStrategy/interfaces/IGsmFeeStrategyFactory.sol

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.0;
+
+interface IGsmFeeStrategyFactory {
```

Comment:
Let's correct all functions' and events' name to be consistent with the new name. I d suggest `IFixedFeeStrategyFactory`

Reply:
Done - within the context of the GSM, I kept it as GsmFeeStrategyFactory. Lemme know if I should change it there too, and I will

Reply:
I think it's important to add "Fixed" because we could have other kind of strategies in the future.

Reply:
Done